cmd/dbhashsum/dbhashsum.go

```diff
@@ -6,13 +6,22 @@ import (
 
 	"github.com/rclone/rclone/backend/dropbox"
 	"github.com/rclone/rclone/cmd"
+	"github.com/rclone/rclone/fs/config/flags"
 	"github.com/rclone/rclone/fs"
 	"github.com/rclone/rclone/fs/operations"
 	"github.com/spf13/cobra"
 )
 
+var (
+	outputBase64 = false
+	downloadFlag = false
+)
+
 func init() {
 	cmd.Root.AddCommand(commandDefinition)
+	cmdFlags := commandDefinition.Flags()
```

Comment:
Gah! We depracated dbhashsum some time ago - perhaps it is time to kill it now, rather than updating it!

Reply:
We'd better drop `dbhashsum` in a dedicated patch for transparency.
Right now let's add the line `dbhashsum will be removed from rclone` in the docs.

Reply:
dbhashsum is retained for now.